src/components/VueTable.vue

```diff
@@ -1311,8 +1311,9 @@ export default {
           }
           this.$set(this.tbodyData[rowIndex][header], 'active', false);
           this.removeClass(['rectangleSelection']);
+          const currentlyEditingCell = this.tbodyData[rowIndex][header].show;
```

Comment:
It work's, but i think it's better to put this condition on the top of the function. I need to look that. I'll come back to you when i look that in deepness. I'll try to look at that on the afternoon !

Reply:
Put your condition like this =>

```
const currentlyEditingCell = this.tbodyData[rowIndex][header].show;

if (!currentlyEditingCell) {
  if (!this.setFirstCell) {
    ...
  }
  ...
  if (this.keys[16]) {
    ...
  } else if (!this.lastSelectOpen && event.keyCode !== 8) {
    ...
  }
}
```

Reply:
Ok i've updated this